src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -48,4 +55,16 @@ export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
   );
 };
 
+// TODO: this was how CampAttendance looked before we deleted it
+// const CampAttendance: FC<PropsType> = ({ attendees }) => {
+//   return attendees > 0 ? (
+//       <div className="camp-venue-people-container">
+//         <div className="camp-venue-people">{attendees}</div>
+//         <FontAwesomeIcon icon={faUser} />
+//       </div>
+//   ) : (
+//       <></>
+//   );
+// };
```

Comment:
- [ ] [TODO] this was how `CampAttendance` looked before we deleted it. Do we need anything here? Can we simplify this?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/881676de95fe02cb743898b3bff51818292514fa